projects/find/find.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+import argparse
+
+"""This method is created to find all files and directories, given a specific path. Optionally, your can use -name flag
+to search only the files that match to a specific name"""
+
+
```

Comment:
there should not be blank lines between method documentation and the actual implementation